src/frontend/sider/Fagsak/Fagsaklinje/Behandlingsmeny/Behandlingsmeny.tsx

```diff
@@ -4,16 +4,15 @@ import styled from 'styled-components';
 
 import { ChevronDownIcon } from '@navikt/aksel-icons';
 import { Button, Dropdown } from '@navikt/ds-react';
-import { RessursStatus } from '@navikt/familie-typer';
 
 import MenyvalgFagsak from './MenyvalgFagsak';
 import MenyvalgBehandling from './OpprettFagsak/MenyvalgBehandling';
-import { BehandlingStatus } from '../../../../typer/behandling';
+import { BehandlingStatus, type IBehandling } from '../../../../typer/behandling';
 import type { IMinimalFagsak } from '../../../../typer/fagsak';
-import { useBehandlingContext } from '../../Behandling/context/BehandlingContext';
 
 interface IProps {
     minimalFagsak: IMinimalFagsak;
+    åpenBehandling?: IBehandling;
```

Comment:
Kan kanskje bare kalle det for `behandling`?

```suggestion
    behandling?: IBehandling;
```